From 19b8051f791f74ed46a51fff8c74a51bf11dd23d Mon Sep 17 00:00:00 2001
From: Kevin Wolf <kwolf@redhat.com>
Date: Tue, 11 Feb 2014 16:24:44 +0100
Subject: [PATCH 27/28] Documentation: qemu-img: Mention SIGUSR1 progress report

RH-Author: Kevin Wolf <kwolf@redhat.com>
Message-id: <1392135884-10508-4-git-send-email-kwolf@redhat.com>
Patchwork-id: 57224
O-Subject: [RHEL-7.0 qemu-kvm PATCH 3/3] Documentation: qemu-img: Mention SIGUSR1 progress report
Bugzilla: 997878
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

Document the SIGUSR1 behaviour of qemu-img. Also, added compare to the
list of subcommands that support -p.

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
Reviewed-by: Benoit Canet <benoit@irqsave.net>
(cherry picked from commit 0e3bd9932f862c1c1e4926939b4d0c602ce214ef)

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
---
 qemu-img.texi | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 qemu-img.texi |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/qemu-img.texi b/qemu-img.texi
index d75a780..7c28759 100644
--- a/qemu-img.texi
+++ b/qemu-img.texi
@@ -53,7 +53,9 @@ indicates that target image must be compressed (qcow format only)
 @item -h
 with or without a command shows help and lists the supported formats
 @item -p
-display progress bar (convert and rebase commands only)
+display progress bar (compare, convert and rebase commands only).
+If the @var{-p} option is not used for a command that supports it, the
+progress is reported when the process receives a @code{SIGUSR1} signal.
 @item -q
 Quiet mode - do not print any output (except errors). There's no progress bar
 in case both @var{-q} and @var{-p} options are used.
-- 
1.7.1

